internal/wasmldr/src/config.rs

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+pub struct Config {
+    pub files: Option<Vec<File>>,
+}
+
+#[derive(Deserialize, Debug)]
+pub struct File {
+    #[serde(rename = "type")]
+    pub type_: String,
```

Comment:
@bstrie I would not want to go through bike shedding over this... But if you really care..

Reply:
I don't really care very much, but in future contexts it may be easier to pick a non-colliding word rather than contorting the code to work around it.

Reply:
"easier" ... well it's just one line added and tab completion does the rest